src/easy/handler.rs

```diff
@@ -3128,6 +3128,20 @@ impl<H> Easy2<H> {
             Ok(list::from_raw(list))
         }
     }
+    /// Get the last http version number
+    ///
+    /// Corresponds to `CURLINFO_HTTP_VERSION` and may return an error if the
+    /// option isn't supported.
+    pub fn response_http_version(&self) -> Result<HttpVersion, Error> {
```

Comment:
This function ideally would be called `http_version` to match the curl constant, but perhaps instead should be called `get_http_version`. We want the names to match the underlying curl names as much as possible, even if it is less understandable.

Reply:
Note that `get_` prefixes are discouraged in Rust, but let's just use it here.